### ISSUE TITLE
[BUG] Windows Hebrew and Greek language packages are not properly displayed in the list of installed pakages

### ISSUE THREAD

**Issue by saimench** [marticliment/UniGetUI#2409]
### Please confirm these before moving forward

- [X] I have searched for my issue and have not found a work-in-progress/duplicate/resolved issue.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
- [X] I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions.
- [X] This issue is about a bug (if it is not, please use the [correct template](https://github.com/marticliment/WingetUI/issues/new/choose)).

### UniGetUI Version

3.1.0

### Windows version, edition and architecture

Windows 10 Version 22H2 x64

### Describe your issue

Windows Hebrew and Greek language packages are not properly displayed in the list of installes packages.

### Steps to reproduce the issue

Searching for the installed packages shows the described bug.

### UniGetUI Log

```text
[03.07.2024 15:33:08] 
                         __  __      _ ______     __  __  ______
                        / / / /___  (_) ____/__  / /_/ / / /  _/
                       / / / / __ \/ / / __/ _ \/ __/ / / // /  
                      / /_/ / / / / / /_/ /  __/ /_/ /_/ // /   
                      \____/_/ /_/_/\____/\___/\__/\____/___/   
                          Welcome to UniGetUI Version 3.1.0
[03.07.2024 15:33:08]
[03.07.2024 15:33:08] Version Code:  3,1
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\AlreadyWarnedAboutNameChange
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\ChocolateyAddedToPath
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\CurrentSessionToken
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\Disable.Net Tool
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\DisableNpm
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\DisablePip
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\DisableScoop
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\IgnoredPackageUpdates.json
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\OldWindowGeometry
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\OperationHistory
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\PreferredLanguage
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\PreferredTheme
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\ShownSearchGuideArrow
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\ShownWelcomeWizard
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\SidepanelWidthBundlesPage
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\SidepanelWidthDiscoverPage
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\SidepanelWidthInstalledPage
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\SidepanelWidthUpdatesPage
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\UpdatesCheckInterval
[03.07.2024 15:33:10] Copying C:\Users\simon\.wingetui\WingetUI_1710338909.926502
[03.07.2024 15:33:10] Deleting old Chocolatey directory C:\Users\simon\.wingetui
[03.07.2024 15:33:10] Using bundled GSudo at C:\Program Files\WingetUI\Assets\Utilities\gsudo.exe
[03.07.2024 15:33:11] Loaded language locale: de
[03.07.2024 15:33:13] Randomly-generated background API auth token for the current session: o9exi8wwfvrmhkh3rd0x78vxwa5xnlxs84hxcxnu689ab0jwys95cxwmvzkyj1vf
[03.07.2024 15:33:14] Api running on http://localhost:7058
[03.07.2024 15:33:15] Moving Bundled Chocolatey from old path to new path...
[03.07.2024 15:33:15] An error occurred while migrating chocolatey
[03.07.2024 15:33:15] System.ArgumentException: The path is empty. (Parameter 'relativeTo')
                         at System.IO.Path.GetRelativePath(String relativeTo, String path, StringComparison comparisonType)
                         at UniGetUI.PackageEngine.Managers.ChocolateyManager.Chocolatey.LoadManager() in C:\SomePrograms\WingetUI-Store\src\UniGetUI.PackageEngine.Managers.Chocolatey\Chocolatey.cs:line 278
[03.07.2024 15:33:15] Lang files were updated successfully from GitHub
[03.07.2024 15:33:16] UniGetUI is up to date
[03.07.2024 15:33:16] The restored geometry did not have exactly 5 items (found length was 1)
[03.07.2024 15:33:16] Command npm was not found on the system
[03.07.2024 15:33:16] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: Npm
                      █ Enabled: False
                      █ THE MANAGER IS DISABLED
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[03.07.2024 15:33:16] Downloaded new icons and screenshots successfully!
[03.07.2024 15:33:16] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: .NET Tool
                      █ Enabled: False
                      █ THE MANAGER IS DISABLED
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[03.07.2024 15:33:17] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: Pip
                      █ Enabled: False
                      █ THE MANAGER IS DISABLED
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[03.07.2024 15:33:18] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: Winget
                      █ Enabled: True
                      █ Found: True
                      █ Fancye exe name: winget.exe
                      █ Executable path: C:\Users\simon\AppData\Local\Microsoft\WindowsApps\winget.exe
                      █ Call arguments: 
                      █ Version: 
                      █   Naive WinGet CLI Version: v1.8.1791
                      █   Microsoft.WinGet.Client PSModule version: 
                      █   Using Native WinGet helper (COM Api)
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[03.07.2024 15:33:18] Adding chocolatey to path since it was not on path.
[03.07.2024 15:33:21] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: Chocolatey
                      █ Enabled: True
                      █ Found: True
                      █ Fancye exe name: choco.exe
                      █ Executable path: C:\Users\simon\AppData\Local\UniGetUI\Chocolatey\choco.exe
                      █ Call arguments: 
                      █ Version: 
                      █   2.2.2
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[03.07.2024 15:33:21] Command scoop was not found on the system
[03.07.2024 15:33:21] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: Scoop
                      █ Enabled: False
                      █ THE MANAGER IS DISABLED
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[03.07.2024 15:33:22] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: PowerShell
                      █ Enabled: True
                      █ Found: True
                      █ Fancye exe name: powershell.exe
                      █ Executable path: C:\WINDOWS\system32\windowspowershell\v1.0\powershell.exe
                      █ Call arguments:  -NoProfile -Command
                      █ Version: 
                      █   Name                           Value                                                                                   
                      █   ----                           -----                                                                                   
                      █   PSVersion                      5.1.19041.4597                                                                          
                      █   PSEdition                      Desktop                                                                                 
                      █   PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                 
                      █   BuildVersion                   10.0.19041.4597                                                                         
                      █   CLRVersion                     4.0.30319.42000                                                                         
                      █   WSManStackVersion              3.0                                                                                     
                      █   PSRemotingProtocolVersion      2.3                                                                                     
                      █   SerializationVersion           1.1.0.1
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[03.07.2024 15:33:24] Found 0 installed packages from Chocolatey
[03.07.2024 15:33:24] Found 0 available updates from Chocolatey
[03.07.2024 15:33:24] Dependency WinGet PowerShell Module for manager Winget is present
[03.07.2024 15:33:24] LoadComponentsAsync finished executing. All managers loaded. Proceeding to interface.
[03.07.2024 15:33:27] Found 1 installed packages from PowerShell
[03.07.2024 15:33:28] Found 0 available updates from PowerShell
[03.07.2024 15:33:49] Found 4 available updates from Winget
[03.07.2024 15:33:51] Found 577 installed packages from Winget
[03.07.2024 15:34:04] Found 0 installed packages from Chocolatey
[03.07.2024 15:34:05] Found 1 installed packages from PowerShell
[03.07.2024 15:34:16] Found 577 installed packages from Winget
[03.07.2024 15:34:43] Update
[03.07.2024 15:34:43] Non-MSStore WinGet Native Icons have been forcefully disabled on code
[03.07.2024 15:34:43] Icon for package SomePythonThings.WingetUIStore.Pre-release was not found, returning default icon
[03.07.2024 15:34:43] Found 0 screenshots for package Id=SomePythonThings.WingetUIStore.Pre-release
[03.07.2024 15:34:45] Loaded details for package SomePythonThings.WingetUIStore.Pre-release on manager Winget
[03.07.2024 15:35:02] Non-MSStore WinGet Native Icons have been forcefully disabled on code
[03.07.2024 15:35:02] Icon for package SomePythonThings.WingetUIStore.Pre-release was not found, returning default icon
[03.07.2024 15:35:35] Uninstall
[03.07.2024 15:35:35] Non-MSStore WinGet Native Icons have been forcefully disabled on code
[03.07.2024 15:35:35] Icon for package MSIX\Microsoft.LanguageExperiencePackhe-IL_19041.61.213.0_neutral__8wekyb3d8bbwe was not found, returning default icon
[03.07.2024 15:35:35] Found 0 screenshots for package Id=MSIX\Microsoft.LanguageExperiencePackhe-IL_19041.61.213.0_neutral__8wekyb3d8bbwe
[03.07.2024 15:35:35] Loaded details for package MSIX\Microsoft.LanguageExperiencePackhe-IL_19041.61.213.0_neutral__8wekyb3d8bbwe on manager Winget
[03.07.2024 15:36:03] Uninstall
[03.07.2024 15:36:03] Non-MSStore WinGet Native Icons have been forcefully disabled on code
[03.07.2024 15:36:03] Icon for package MSIX\Microsoft.LanguageExperiencePackhe-IL_19041.61.213.0_neutral__8wekyb3d8bbwe was not found, returning default icon
[03.07.2024 15:36:03] Found 0 screenshots for package Id=MSIX\Microsoft.LanguageExperiencePackhe-IL_19041.61.213.0_neutral__8wekyb3d8bbwe
[03.07.2024 15:45:19] Uninstall
[03.07.2024 15:45:19] Non-MSStore WinGet Native Icons have been forcefully disabled on code
[03.07.2024 15:45:19] Icon for package MSIX\Microsoft.LanguageExperiencePackel-GR_19041.78.252.0_neutral__8wekyb3d8bbwe was not found, returning default icon
[03.07.2024 15:45:19] Found 0 screenshots for package Id=MSIX\Microsoft.LanguageExperiencePackel-GR_19041.78.252.0_neutral__8wekyb3d8bbwe
[03.07.2024 15:45:19] Loaded details for package MSIX\Microsoft.LanguageExperiencePackel-GR_19041.78.252.0_neutral__8wekyb3d8bbwe on manager Winget
```

### Package Managers Logs

```text
Manager Winget with version:
Naive WinGet CLI Version: v1.8.1791
Microsoft.WinGet.Client PSModule version: 
Using Native WinGet helper (COM Api)

——————————————————————————————————————————


Logged native task on manager Winget. Task type is ListSources
Process start time: 03.07.2024 15:33:18
Process end time:   03.07.2024 15:33:18

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\simon\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 03.07.2024 15:33:22
Process end time:   03.07.2024 15:33:35

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is ListPackages
Subprocess executable: "C:\WINDOWS\system32\windowspowershell\v1.0\powershell.exe"
Command-line arguments: "-ExecutionPolicy Bypass -NoLogo -NoProfile"
Process start time: 03.07.2024 15:33:22
Process end time:   03.07.2024 15:33:51

-- Process STDIN
 ...

-- Process STDOUT
 ...

-- Process STDERR
  Write-Output : Die Benennung "﻿Write-Output" wurde nicht als Name eines Cmdlet, einer Funktion, einer Skriptdatei 
  oder eines ausf�hrbaren Programms erkannt. �berpr�fen Sie die Schreibweise des Namens, oder ob der Pfad korrekt ist 
  (sofern enthalten), und wiederholen Sie den Vorgang.
  In Zeile:1 Zeichen:1
  + ﻿Write-Output (Get-Module -Name Microsoft.WinGet.Client).Version
  + ~~~~~~~~~~~~~~~
      + CategoryInfo          : ObjectNotFound: (﻿Write-Output:String) [], CommandNotFoundException
      + FullyQualifiedErrorId : CommandNotFoundException
   

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListSources
Process start time: 03.07.2024 15:33:23
Process end time:   03.07.2024 15:33:23

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is ListUpdates
Subprocess executable: "C:\WINDOWS\system32\windowspowershell\v1.0\powershell.exe"
Command-line arguments: "-ExecutionPolicy Bypass -NoLogo -NoProfile"
Process start time: 03.07.2024 15:33:35
Process end time:   03.07.2024 15:33:49

-- Process STDIN
 ...

-- Process STDOUT
 ...

-- Process STDERR
  Write-Output : Die Benennung "﻿Write-Output" wurde nicht als Name eines Cmdlet, einer Funktion, einer Skriptdatei 
  oder eines ausf�hrbaren Programms erkannt. �berpr�fen Sie die Schreibweise des Namens, oder ob der Pfad korrekt ist 
  (sofern enthalten), und wiederholen Sie den Vorgang.
  In Zeile:1 Zeichen:1
  + ﻿Write-Output (Get-Module -Name Microsoft.WinGet.Client).Version
  + ~~~~~~~~~~~~~~~
      + CategoryInfo          : ObjectNotFound: (﻿Write-Output:String) [], CommandNotFoundException
      + FullyQualifiedErrorId : CommandNotFoundException
   

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is ListPackages
Subprocess executable: "C:\WINDOWS\system32\windowspowershell\v1.0\powershell.exe"
Command-line arguments: "-ExecutionPolicy Bypass -NoLogo -NoProfile"
Process start time: 03.07.2024 15:34:02
Process end time:   03.07.2024 15:34:16

-- Process STDIN
 ...

-- Process STDOUT
 ...

-- Process STDERR
  Write-Output : Die Benennung "﻿Write-Output" wurde nicht als Name eines Cmdlet, einer Funktion, einer Skriptdatei 
  oder eines ausf�hrbaren Programms erkannt. �berpr�fen Sie die Schreibweise des Namens, oder ob der Pfad korrekt ist 
  (sofern enthalten), und wiederholen Sie den Vorgang.
  In Zeile:1 Zeichen:1
  + ﻿Write-Output (Get-Module -Name Microsoft.WinGet.Client).Version
  + ~~~~~~~~~~~~~~~
      + CategoryInfo          : ObjectNotFound: (﻿Write-Output:String) [], CommandNotFoundException
      + FullyQualifiedErrorId : CommandNotFoundException
   

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is LoadPackageDetails
Process start time: 03.07.2024 15:34:43
Process end time:   03.07.2024 15:34:45

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged native task on manager Winget. Task type is LoadPackageVersions
Process start time: 03.07.2024 15:34:43
Process end time:   03.07.2024 15:34:44

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged native task on manager Winget. Task type is LoadPackageDetails
Process start time: 03.07.2024 15:35:35
Process end time:   03.07.2024 15:35:35

-- Task errors
  Failed to get catalog Lokaler PC. Is the package local?

The task reported a failure (1)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is LoadPackageVersions
Process start time: 03.07.2024 15:35:35
Process end time:   03.07.2024 15:35:35

-- Task errors
  Failed to get catalog Lokaler PC. Is the package local?

The task reported a failure (1)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is LoadPackageVersions
Process start time: 03.07.2024 15:36:03
Process end time:   03.07.2024 15:36:03

-- Task errors
  Failed to get catalog Lokaler PC. Is the package local?

The task reported a failure (1)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is LoadPackageDetails
Process start time: 03.07.2024 15:45:19
Process end time:   03.07.2024 15:45:19

-- Task errors
  Failed to get catalog Lokaler PC. Is the package local?

The task reported a failure (1)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is LoadPackageVersions
Process start time: 03.07.2024 15:45:19
Process end time:   03.07.2024 15:45:19

-- Task errors
  Failed to get catalog Lokaler PC. Is the package local?

The task reported a failure (1)

——————————————————————————————————————————
```

### Screenshots and videos

![image](https://github.com/marticliment/UniGetUI/assets/5787497/80106edf-ed29-49ff-b96a-4a2366484e61)
![image](https://github.com/marticliment/UniGetUI/assets/5787497/8ecb119a-70f8-4b99-b291-37f35759706b)
![image](https://github.com/marticliment/UniGetUI/assets/5787497/3c113eda-ce78-4f05-a454-aef134c104dc)

**Comment by marticliment** [comment]
duplicate of #2242

**Comment by marticliment** [comment]
Hello,

To all users facing issues with version 3.1.0, please try the new 3.1.1 beta, which aims to fix this issues.

UniGetUI 3.1.1-beta0 release: https://github.com/marticliment/UniGetUI/releases/tag/3.1.1-beta0
Direct installer URL: https://github.com/marticliment/UniGetUI/releases/download/3.1.1-beta0/UniGetUI.Installer.exe

Excuse me for the inconveniences.

**Comment by saimench** [comment]
The bug is fixed in UniGetUI 3.1.1! Thanks!
![image](https://github.com/user-attachments/assets/286570bd-7bfa-41d1-b4cd-1182d4d9754e)
